src/storage/storageResolver.js

```diff
@@ -62,18 +62,18 @@ function storageRangeInternal (self, slotKey, tx, stepIndex, address, callback)
   if (cached && cached.storage[slotKey]) { // we have the current slot in the cache and maybe the next 1000...
     return callback(null, cached.storage)
   }
-  storageRangeWeb3Call(tx, address, slotKey, self.maxSize, (error, storage, complete) => {
+  storageRangeWeb3Call(tx, address, slotKey, self.maxSize, (error, storage, nextKey) => {
     if (error) {
       return callback(error)
     }
-    if (!storage[slotKey]) {
+    if (!storage[slotKey] && slotKey !== zeroSlot) { // we don't cache the zero slot (could lead to inconsistency)
```

Comment:
Because the storage actually stores only hashed key now and `zeroSlot` is obviously not a hashed key.